jmespath-core/src/main/java/io/burt/jmespath/jcf/JcfRuntime.java

```diff
@@ -160,4 +173,80 @@ public Object createNumber(double n) {
   public Object createNumber(long n) {
     return n;
   }
+
+  /**
+   * Helper method to render a value as JSON.
+   *
+   * Assumes that <code>null</code>, <code>number</code> and <code>boolean</code>
+   * render themseves correctly with <code>toString</code>, and that
+   * <code>string</code> renders itself as an unquoted string.
+   */
+  private String unparse(Object object) {
+    switch (typeOf(object)) {
+      case NUMBER:
+        return unparseNumber(object);
+      case BOOLEAN:
+        return unparseBoolean(object);
+      case NULL:
+        return unparseNull(object);
+      case STRING:
+        return unparseString(object);
```

Comment:
I think you might as well go one step further and inline the implementations of number, boolean, null, and string. They are all one-liners and I don't think it makes sense to keep them as their own methods now when they can't be overridden anymore.